neural_style.py

```diff
@@ -10,15 +10,25 @@
 import math
 from argparse import ArgumentParser
 
+from matplotlib import pyplot as plt
+
+import tensorflow as tf
```

Comment:
This is part of a bigger discussion about where the luminosity transfer should happen, since tf is used to perform that